src/griddyn/solvers/arkodeInterface.h

```diff
@@ -62,6 +62,9 @@ class arkodeInterface : public sundialsInterface
     virtual void set (const std::string &param, const std::string &val) override;
     virtual void set (const std::string &param, double val) override;
     virtual double get (const std::string &param) const override;
+
+    virtual void updateMaxIterations() override;
```

Comment:
why is this a standalone function vs the "set" function?

Reply:
No particular reason; I can move it to the set function if you want.

The only real advantage is having to say "maxiterations" as the parameter key once, instead of once per solver.

Reply:
I think my preference would be to avoid adding an additional virtual function call.  Otherwise there is no clear boundary as to when to stop adding them.

Reply:
Fixed:
 * in the sundials implementations, set happens in-place, and calls the appropriate iteration setting functions
 * otherwise, it falls through to solverInterface.cpp, where it just sets `max_iterations` for the other two solvers to use